### DISCUSSION TITLE
Is it possible to remove the TrayIcon Menu scroll button, or is this a bug?

### DISCUSSION THREAD

**Discussion by xD0135** [AvaloniaUI/Avalonia#11373] [Q&A] [answered]
Is there a way to remove the scroll button indicator in the TrayIcon Menu? Or is this a bug?

This simple example with a single menu item still includes a clickable scroll button (arrow pointing down), and I'm wondering if there's a way to remove it, or control when to include it:
```c#
<TrayIcon.Icons>
        <TrayIcons>
            <TrayIcon Icon="/Assets/avalonia-logo.ico" ToolTipText="Avalonia Tray Icon ToolTip">
                <TrayIcon.Menu>
                    <NativeMenu>
                        <NativeMenuItem Header="Show/Hide" />
                    </NativeMenu>
                </TrayIcon.Menu>
            </TrayIcon>
        </TrayIcons>
    </TrayIcon.Icons>
``` 

![Screenshot 2023-05-14 145750](https://github.com/AvaloniaUI/Avalonia/assets/32652329/ef323529-cf83-45f2-a1be-3fa1b6a95c29)

Tested using Avalonia.Desktop 11.0.0-preview8 on Windows 11. Thanks,.

**Comment by timunie** [comment]
I'm not able to reproduce it using current nightly: 
![image](https://github.com/AvaloniaUI/Avalonia/assets/47110241/4783bfd4-bf9e-45bd-8807-fd198738ca9e)

Is it possible that you have a custom theme applied?

**Comment by xD0135** [comment]
Thanks for the reply @timunie. My sample app is really bare bones:

```xml
<Application
    x:Class="Sample.App"
    xmlns="https://github.com/avaloniaui"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:Sample"
    RequestedThemeVariant="Default">
    <!--  "Default" ThemeVariant follows system theme variant. "Dark" or "Light" are other available options.  -->

    <Application.DataTemplates>
        <local:ViewLocator />
    </Application.DataTemplates>

    <Application.Styles>
        <FluentTheme />
    </Application.Styles>

    <TrayIcon.Icons>
        <TrayIcons>
            <TrayIcon Icon="/Assets/avalonia-logo.ico" ToolTipText="Avalonia Tray Icon ToolTip">
                <TrayIcon.Menu>
                    <NativeMenu>
                        <NativeMenuItem Header="Show/Hide" />
                    </NativeMenu>
                </TrayIcon.Menu>
            </TrayIcon>
        </TrayIcons>
    </TrayIcon.Icons>

</Application>
```
You mentioned that you tested in nightly. Could you also verify on 11.0.0-preview8? Hopefully it's only impacting this version.

**Reply by rabbitism** [reply]
I'm not able to reproduce either. any special settings? like scaling?

**Reply by xD0135** [reply]
I don't think so. This sample app was created as a cross-platform app by following this guide: https://docs.avaloniaui.net/tutorials/developing-for-mobile/create-a-cross-platform-solution, and after that the only change I did was to add the tray icon. I also just tried creating a single-OS app for Windows and adding the above tray icon, and still see the issue😿.

Any other thoughts on how I can further debug this?

**Reply by rabbitism** [reply]
![image](https://github.com/AvaloniaUI/Avalonia/assets/14807942/a20b0273-f8f0-4088-a7c1-bce3f4756187)
This is what it look like in my machine. preview8.

**Reply by timunie** [reply]
> Any other thoughts on how I can further debug this?

@xD0135 still I believe attaching the sln you have could be useful! Sometimes it is just a single line anywhere that no one has seen before. 

Also make sure that your Win11 is up to date. I had recently an issue where Win11 had broken all my context menus in File-Explorer. This seems to be fixed now. Probably it's related to your issue.

**Reply by xD0135** [reply]
@timunie here's the snl from a single-OS sample app that also has the issue:

```
Microsoft Visual Studio Solution File, Format Version 12.00
# Visual Studio Version 17
VisualStudioVersion = 17.6.33706.43
MinimumVisualStudioVersion = 10.0.40219.1
Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AvaloniaApplication1", "AvaloniaApplication1.csproj", "{1593D70C-1784-48D5-94F5-11D1C35708B4}"
EndProject
Global
	GlobalSection(SolutionConfigurationPlatforms) = preSolution
		Debug|Any CPU = Debug|Any CPU
		Release|Any CPU = Release|Any CPU
	EndGlobalSection
	GlobalSection(ProjectConfigurationPlatforms) = postSolution
		{1593D70C-1784-48D5-94F5-11D1C35708B4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{1593D70C-1784-48D5-94F5-11D1C35708B4}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{1593D70C-1784-48D5-94F5-11D1C35708B4}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{1593D70C-1784-48D5-94F5-11D1C35708B4}.Release|Any CPU.Build.0 = Release|Any CPU
	EndGlobalSection
	GlobalSection(SolutionProperties) = preSolution
		HideSolutionNode = FALSE
	EndGlobalSection
	GlobalSection(ExtensibilityGlobals) = postSolution
		SolutionGuid = {214DE6F9-1CFF-46D5-9216-D8F4F226C7A9}
	EndGlobalSection
EndGlobal
```

Regarding keeping Windows 11 up to date, yes, it's on the latest:
```
Edition		Windows 11 Pro
Version		21H2
OS build	22000.1936
Experience	Windows Feature Experience Pack 1000.22001.1000.0
```

**Reply by timunie** [reply]
@xD0135 I mean the entire solution, not just that file. We need something to reproduce it on our side. 

Different idea can be related to what @MrJul wrote below.

**Reply by rabbitism** [reply]
> @xD0135 I mean the entire solution, not just that file. We need something to reproduce it on our side. 
> 
> Different idea can be related to what @MrJul wrote below.

Hi Tim, i also found the arrow always show in semi.avalonia demo when released with native aot. Did you try other themes?

**Reply by timunie** [reply]
@rabbitism interesting. Only when released with Native AOT? What is your DPI Scale?

**Reply by rabbitism** [reply]
Yes normal dpi and scaling. I can imagine there are some rounding issue in the visibility converter but i didnt have time to look at it.

**Comment by MrJul** [comment, marked as answer]
Is your screen at 125% DPI? If yes, this is probably #9501

**Reply by xD0135** [reply]
Interesting find @MrJul! My screen was at 105% and resetting to 100% I can confirm the arrow did not show up. Seeing the update from @rabbitism above, and #9501, this issue appears to be a bug then. I'm glad it wasn't just me 😺 .

**Reply by timunie** [reply]
marked as answer as the link to the issue is the correct answer in this case.
